docs/ingestion/kafka-ingestion.md

```diff
@@ -113,10 +118,13 @@ The following example shows a supervisor spec for the Kafka indexing service:
 
 </details>
 
-### I/O configuration
+### `ioConfig`
+
+The `ioConfig` influences how data is read from a source system, such as Apache Kafka, Amazon S3, a mounted filesystem, or any other supported source system.
 
-The following table outlines the `ioConfig` configuration properties specific to Kafka.
-For configuration properties shared across all streaming ingestion methods, refer to [Supervisor I/O configuration](supervisor.md#io-configuration).
+The following table outlines the `ioConfig` configuration properties specific to Apache Kafka.
+
+For configuration properties shared across all supervisor-led ingestion, refer to [supervisor `ioConfig`](supervisor.md#ioconfig).
```

Comment:
I wonder if "streaming ingestion methods" is more suitable (easier to comprehend) than "supervisor-led ingestion".

Reply:
Maybe:
> common to all streaming ingestion supervisor configurations.